forms/CommentCreateForm.php

```diff
@@ -67,9 +70,11 @@ public function save()
         $Comment = $this->Comment;
 
         if (empty($this->id)) {
-            $Comment = new Comments\models\Comment();
+            $Comment =  \Yii::$app->getModule(Comments\Module::NAME)->model('comment');
```

Comment:
Do you mean to use makeObject in each definition (or get, that would be the same in this case) to make the users set new Dependency injections before calling the system in case they want changes?

Like:
```php
$container->set('rmrevin\yii\module\Comments\models\Comment', 'new\model\Comment');
echo \rmrevin\yii\module\Comments\widgets\CommentListWidget::widget([
            'entity' => 'blah-15', // type and id
        ]);
```